app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/Filter.kt

```diff
@@ -95,8 +96,8 @@ fun FilterPanel(
     parkingViewModel: ParkingViewModel,
     displayHeader: Boolean,
     addressViewModel: AddressViewModel,
-    myLocation: MutableState<Boolean> = mutableStateOf(false),
-    chosenLocation: MutableState<Address> =
+    myLocation: State<Boolean> = mutableStateOf(false),
```

Comment:
Can you explain your choice to go from MutableState to State ?

Reply:
The idea was that since I refactored chosen location and my location they were state and not mutable state hence i did the modification so I could pass them as arguments

Reply:
I'm convinced by this explanation